configs/xcache-consistency-check/xcache-consistency-check/default.cfg

```diff
@@ -0,0 +1,17 @@
+[Main]
+
+#(REQUIRED) Where are the .root files to analyze
+#path = /xcache-root/
```

Comment:
Can't we determine this by the `oss.localroot` and/or exports of the XCache config?

Reply:
Does xrootd provide a way to query config or parse config files? I suggest not trying to parse xrootd config files ourselves.

Reply:
Yes but it is messy. YOu can use the `cconfig` tool and you would pass it lets say the atlas or  CMS config file:

`cconfig -c /etc/xrootd/xrootd-cms-xcache.cfg`

And a different one for ATlas. Moreover it does not read enviroment variables which we use to configure most of the caches

Reply:
Ah, `cconfig` not reading env vars is a dealbreaker. If we can't figure the paths out programmatically, let's flesh out the comment, e.g. if the value of `oss.localroot` in the XCache config would be reasonable to use, let's suggest that.